src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const thumbGallery = document.querySelectorAll('.gallery__thumb');
+
+thumbGallery.forEach(item =>
+  item.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const urlImage = e.path[1].getAttribute('href');
```

Comment:
Event.path - was deprecated and removed. https://chromestatus.com/feature/5726124632965120

Refactor your implementation